src/pfe/portal/modules/Project.js

```diff
@@ -65,6 +64,17 @@ module.exports = class Project {
 
     // locOnDisk is used by the UI and needs to match what it sees.
     if (args.locOnDisk) this.locOnDisk = args.locOnDisk;
+
```

Comment:
Might want to consider `isWindowsPath`ing the `pathToMonitor` from args to change any existing bad `pathToMonitors` - is this impossible?

Reply:
This is impossible as pathToMonitor isn't there at the moment.